modules/coverage-report/src/test/java/org/jooby/issues/Issue616.java

```diff
@@ -9,8 +9,8 @@
   public static class Person {
 
     // mandatory fields
-    private final String name;
-    private final String surname;
+    private String name = null;
```

Comment:
why these changes?

Reply:
This was because of the Mock init needs a default constructor, else it won't init the corresponding test. But I could be wrong.